Automattic/sensei
# Add usage tracking step styles

Fixes #5645

Based on https://github.com/Automattic/sensei/pull/5694

### Changes proposed in this Pull Request

* Add usage tracking styles.

The usage tracking step will have a new iteration in a new PR after a talk with Andrei about the position and round borders. More details: p1663369234771779-slack-C013QUH20TS

### Testing instructions


* Navigate to the usage tracking step (`/wp-admin/admin.php?page=sensei_setup_wizard&step=tracking`), and make sure the styles are working properly.


### Screenshot / Video

https://user-images.githubusercontent.com/876340/190829810-582b121d-4ae0-4612-9bf8-307698419f06.mp4
